pkg/utils/utils.go

```diff
@@ -114,3 +122,25 @@ func RoundUpDuration(t time.Duration) time.Duration {
 	}
 	return t.Round(time.Hour)
 }
+
+func getLogLevel() zapcore.Level {
+	logLevel := os.Getenv(log.LevelEnvName)
+	if logLevel == "" {
+		return defaultLogLevel
+	}
+	level, err := zapcore.ParseLevel(logLevel)
+	if err != nil {
+		return defaultLogLevel
+	}
+	return level
+}
+
+func GetLogger() logr.Logger {
```

Comment:
We shouldn't move this to the utility function. This is not the default logger we use in Kanister.
It would mislead developers.

Reply:
Please check if we can use default logger from `github.com/kanisterio/kanister/pkg/log` instead of creating this new type of logger.

Reply:
Default logger from [github.com/kanisterio/kanister/pkg/log](http://github.com/kanisterio/kanister/pkg/log) ,   does not comply with the log.logger. Anything else I can use here

Reply:
Using `log.SetLogger(logr.New(log.NullLogSink{}))` from `sigs.k8s.io/controller-runtime/pkg/log` package. 

In the default implementation logger is being initialised in controller-runtime by using `NullLogSink` in case of nothing is set explicitly. As we want to keep the behaviour similar to what it used to be earlier and also to get rid of the warning, we use similar approach.